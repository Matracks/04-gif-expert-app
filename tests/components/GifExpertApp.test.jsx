import { render, screen } from "@testing-library/react"
import { GifExpertApp } from "../../src/GifExpertApp"


describe('test in GifExpertApp', () => { 

    test('should match with the snapshot', () => { 

        const { container } = render( <GifExpertApp /> )
        expect(container).toMatchSnapshot()

    })

    test('should be retun initial state', () => { 

        render( <GifExpertApp /> )

        expect( screen.getByText('Cargando...')).toBeTruthy()

    })

})