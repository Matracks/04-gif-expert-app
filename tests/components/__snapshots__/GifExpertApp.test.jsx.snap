// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test in GifExpertApp should match with the snapshot 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="Search Gif"
      type="text"
      value=""
    />
  </form>
  <h2>
    Cargando...
  </h2>
  <div
    class="card-grid"
  />
</div>
`;
